src/app/components/download-listings/download-listings.jsx

```diff
@@ -61,6 +61,7 @@ const allHeaders = [
   { headerName: 'Real World Testing Results URL', objectKey: 'rwtResultsUrl', group: 'rwt' },
   { headerName: 'SVAP', objectKey: 'svap', group: 'svap' },
   { headerName: 'SVAP Notice URL', objectKey: 'svapNoticeUrl', group: 'svap' },
+  { headerName: 'Risk Management Summary Information ', objectKey: 'riskManagementSummaryInformation' },
```

Comment:
This one should be higher up, in front of the SVAP stuff, to match the organization of the category